elpis/engines/kaldi/objects/model.py

```diff
@@ -218,24 +218,57 @@ def train():
                 os.remove(run_log_path)
             stages = os.listdir(local_kaldi_path.joinpath('stages'))
             run(f"touch {run_log_path};")
+
+            # organise stage logs in a dir
+            train_log_dir = self.path.joinpath('train-logs')
+            if os.path.exists(train_log_dir ):
+                shutil.rmtree(train_log_dir )
+            os.mkdir(train_log_dir )
+
+            stage_count = 0
+
             for stage in sorted(stages):
                 print(f"======== STAGE {stage} STARTING ========")
                 self.stage_status = (stage, 'in-progress', '')
-                
+
+                # Create log file
+                stage_log_path = self.path.joinpath(os.path.join(train_log_dir, f'stage_{stage_count}.log'))
+                with open(stage_log_path, 'w+'):
+                    pass
+
+                #  Manipulate stage templates with user-defined settings
                 with open(local_kaldi_path.joinpath('stages').joinpath(stage), 'r') as file :
                     filedata = file.read()
-
+                # Add settings to replace here
```

Comment:
This should eventually be replaced with jinja templates or something similar.

Reply:
have made a note to do this